test/Integration/conv-to-matmul.mlir

```diff
@@ -15,6 +15,15 @@
 // RUN: FileCheck %s -check-prefix=TRANSFORM
 //
 
+// Validate default pipeline
+// RUN: tpp-run %s -print \
+// RUN:  -e entry -entry-point-result=void | \
+// RUN: FileCheck %s -check-prefix=LINALG
+
+// RUN: tpp-run %s -print \
```

Comment:
Not clear to me what's the difference here. The original seems to be between with and without `-rewrite-conv-to-matmul-or-brgemm`, but the default pipeline will not skip that.

I don't know if the original intention can be mapped to lower-to-loops {true, false}.

Reply:
Good catch, in this case it doesn't matter for `tpp-run` as this conv rewrite is always enabled in the default pipeline.
One of these runs can be removed.

Regarding your comment 2) - I'll have to double check them, in some cases there are difference with an extra `tpp-opt` preprocessing but there might be a few that could be cleaned up.
About comment 3) - the tests that don't print/return outputs rely only on `check` dialect validation. But to be sure, I'll check again if that's truly the case in all of them.